app/src/main/java/de/danoeh/antennapod/fragment/ItemFragment.java

```diff
@@ -243,6 +244,7 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
             if (item.hasMedia()) {
                 FeedMedia media = item.getMedia();
                 if (!media.isDownloaded()) {
+                    PlaybackService.startIfNotRunning(getActivity(), media, true, false);
```

Comment:
the true-false combination seems pretty common.
maybe an extra method that only takes activity/context and media and then calls this method?

Reply:
I think I have used this combination in too many places. Resuming a streamed episode after stopping the service is not possible.